collectors/lib/java.py

```diff
@@ -30,20 +30,19 @@
 
 PERIOD = 60
 
-IGNORED_METRICS = set(["revision", "hdfsUser", "hdfsDate", "hdfsUrl", "date",
-                       "hdfsRevision", "user", "hdfsVersion", "url", "version",
-                       "NamenodeAddress", "Version", "RpcPort", "HttpPort",
-                       # These are useless as-is because they represent the
-                       # thread that's dedicated to serving JMX RPCs.
-                       "CurrentThreadCpuTime", "CurrentThreadUserTime",
-                       # List of directories used by the DataNode.
-                       "StorageInfo",
-                       "VolumeInfo",
-                       "ServerName",
-                       "ZookeeperQuorum",
-                       "Verbose",
-                       "Type"
-                      ])
+IGNORED_METRICS = {"revision", "hdfsUser", "hdfsDate", "hdfsUrl", "date",
```

Comment:
this blacklist feels a bit weird to me, we already use an "opt-in" style white list in subclasses to choose metrics to propagate. Should this similarly be a subclass based mechanism?